app/upgrades.go

```diff
@@ -54,15 +54,16 @@ func (app *App) RegisterUpgradeHandlers(experimental bool) {
 	if !app.UpgradeKeeper.IsSkipHeight(upgradeInfo.Height) {
 		if upgradeInfo.Name == planName {
 			storeUpgrades := storetypes.StoreUpgrades{
-				Added: []string{icacontrollertypes.StoreKey, ibcfeetypes.StoreKey},
+				Added: []string{ibcfeetypes.StoreKey},
 			}
 
 			// configure store loader that checks if version == upgradeHeight and applies store upgrades
 			app.SetStoreLoader(upgradetypes.UpgradeStoreLoader(upgradeInfo.Height, &storeUpgrades))
 		}
 		if experimental && upgradeInfo.Name == gravityPlanName {
 			storeUpgrades := storetypes.StoreUpgrades{
-				Added: []string{icacontrollertypes.StoreKey},
+				Added:   []string{ibcfeetypes.StoreKey},
+				Deleted: []string{icacontrollertypes.StoreKey},
```

Comment:
was 0.9.0-alpha already applied? in that case we should change the plan name?

Reply:
bump to `v0.8.0-gravity-alpha3` for now.